cli/lsp/tsc.rs

```diff
@@ -3868,71 +3870,76 @@ struct LoadResponse {
 }
 
 #[op2]
-#[serde]
-fn op_load(
+fn op_load<'s>(
+  scope: &'s mut v8::HandleScope,
   state: &mut OpState,
   #[serde] args: SpecifierArgs,
-) -> Result<Option<LoadResponse>, AnyError> {
+) -> Result<v8::Local<'s, v8::Value>, AnyError> {
```

Comment:
We should return an array with 3 elements here, instead of using an object (lower pressure on GC and faster to serialize because no need to serialize 3 keys as strings)